assets/src/edit-story/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -52,14 +64,56 @@ const StyledCheckbox = styled(Checkbox)`
 `;
 
 function VideoOptionsPanel({ selectedElements, pushUpdate }) {
+  const isMuteVideoEnabled = useFeature('enableMuteVideo');
+  const { isTranscodingEnabled } = useFFmpeg();
+  const { optimizeVideoMuted } = useLocalMedia((state) => ({
+    optimizeVideoMuted: state.actions.optimizeVideoMuted,
+  }));
+  const resource = getCommonValue(selectedElements, 'resource');
+  const { isMuted, isTranscoding, local } = resource;
   const loop = getCommonValue(selectedElements, 'loop');
+  const isSingleElement = selectedElements.length === 1;
+
+  const handleHandleMuteVideo = useCallback(() => {
+    optimizeVideoMuted({ resource });
+  }, [resource, optimizeVideoMuted]);
+
+  const canMuted = useMemo(() => {
+    return (
+      isMuteVideoEnabled &&
+      isTranscodingEnabled &&
+      !local &&
+      !isMuted &&
+      !isTranscoding &&
+      isSingleElement
+    );
+  }, [
+    isMuteVideoEnabled,
+    isTranscodingEnabled,
+    local,
+    isMuted,
+    isTranscoding,
+    isSingleElement,
+  ]);
 
   const checkboxId = `cb-${uuidv4()}`;
   return (
     <SimplePanel
       name="videoOptions"
       title={__('Video Settings', 'web-stories')}
     >
+      {canMuted && (
+        <Row>
+          <StyledButton
+            variant={BUTTON_VARIANTS.RECTANGLE}
+            type={BUTTON_TYPES.SECONDARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={handleHandleMuteVideo}
+          >
+            {__('Muted video', 'web-stories')}
+          </StyledButton>
+        </Row>
+      )}
```

Comment:
ATM, the button is hidden after pressed. This is weird UX. Can we do something better?

Reply:
Changed.